tasks/sep/ink-001-fp-holocene-pectra-upgrade/README.md

```diff
@@ -0,0 +1,44 @@
+# Ink Sepolia Holocene Hardfork + L1 Pectra Support Upgrade
+
+Status: READY TO SIGN
+
+## Objective
+
+Upgrades the Fault Proof contracts for the Holocene hardfork and utilizes the absolute prestate `0x03dfa3b3ac66e8fae9f338824237ebacff616df928cf7dada0e14be2531bc1f4` that enables L1 pectra support.
+
+The proposal was:
```

Comment:
Holocene proposal was approved but L1 pectra support has not yet been approved, so this is slightly misleading